internal/clients/content_sources/client.go

```diff
@@ -0,0 +1,90 @@
+//go:generate go run -mod=mod github.com/deepmap/oapi-codegen/v2/cmd/oapi-codegen --config client.cfg.yaml content-sources.v1.json
+
```

Comment:
Just a note: this file has a go generate comment, but this particular file is NOT generated.

If I can suggest suggest creating an empty `package.go` file with only this line separately to prevent this confusion. I skipped reviewing this file in other review because I believed it was auto-generated... :-D